### PR TITLE
Prepare redirect url

### PR DESCRIPTION
**Pull request by ilsinszkibal**
I've hit a problem around 301 response, providing possible fix.

### CODE REVIEW

File: github/Requester.py [hunk 1]
```diff
@@ -411,10 +411,20 @@ def __requestRaw(self, cnx, verb, url, requestHeaders, input):
             return self.__requestRaw(original_cnx, verb, url, requestHeaders, input)
 
         if status == 301 and 'location' in responseHeaders:
-            return self.__requestRaw(original_cnx, verb, responseHeaders['location'], requestHeaders, input)
+            redirect_url = self.__permanentRedirectURL(responseHeaders['location'], cnx)
```
Comment (ilsinszkibal) [inline github/Requester.py]: The problem that I've seen:
- cnx.protocol: https
- cnx.host: github.enterprise.com
- responseHeaders['location']: https://github.enterprise.com/api/

**Comment by sfdye** [comment]
Are you trying to address #906?

**Comment by ilsinszkibal** [comment]
I didn't find this issue, but yes, this is the same issue, that I have. Any improvement that I need to make @sfdye?

**Comment by sfdye** [comment]
Closing in favor of #916 

@ilsinszkibal Thank you for your PR!

**Pull request closed without merging**